lib/manager/cargo/artifacts.spec.ts

```diff
@@ -67,9 +63,9 @@ describe('manager/cargo/artifacts', () => {
   });
   it('returns null if unchanged', async () => {
     fs.stat.mockResolvedValueOnce({ name: 'Cargo.lock' } as any);
-    fs.readFile.mockResolvedValueOnce('Current Cargo.lock' as any);
+    fs.readLocalFile.mockResolvedValueOnce('Current Cargo.lock' as any);
```

Comment:
```suggestion
    fs.readLocalFile.mockResolvedValueOnce('Current Cargo.lock');
```
Cast not required, please remove on others too